R/geom_signif.R

```diff
@@ -104,6 +105,7 @@ stat_signif <- function(mapping = NULL,
                         vjust = 0,
                         parse = FALSE,
                         manual = FALSE,
+                        FDR=FALSE,
```

Comment:
Instead of creating a new argument for every _p_-value adjustment method, a better option here is to instead introduce argument `method = "none"`. The default would make this addition backward-compatible, and this adjustment becomes relative only when `method` is not equal to `"none"`.